NitroxPatcher/Patches/SubNameInput_OnColorChange_Patch.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Harmony;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.Helper;
+using System;
```

Comment:
Minor: we use VisualStudio's [<kbd>ctrl</kbd>+<kbd>r</kbd>, <kbd>ctrl</kbd>+<kbd>g</kbd>] to sort using statements (this removed unused ones as well).